resources/group.go

```diff
@@ -225,6 +224,22 @@ func (obj *GroupRes) CheckApply(apply bool) (checkOK bool, err error) {
 type GroupUID struct {
 	BaseUID
 	name string
+	gid  *uint32
+}
+
+// IFF aka if and only if they are equivalent, return true. If not, false.
+func (obj *GroupUID) IFF(uid ResUID) bool {
+	res, ok := uid.(*GroupUID)
+	if !ok {
+		return false
+	}
+	var g, n bool
```

Comment:
I'm not sure if this logic is right. First of all, don't we want this to return true if both valid pairs (gid & name) are true, out of whichever are present?

I think the code can be simplified by checking for different and returning false early, only to return true at the end if all is good.